src/core/server/plugins/plugin.test.ts

```diff
@@ -263,3 +262,65 @@ test('`stop` calls `stop` defined by the plugin instance', async () => {
   await expect(plugin.stop()).resolves.toBeUndefined();
   expect(mockPluginInstance.stop).toHaveBeenCalledTimes(1);
 });
+
+describe('#getConfigSchema()', () => {
+  it('reads config schema from plugin', () => {
```

Comment:
__optional nit:__ this file uses `test` instead of `it`, we probably want to be consistent in this PR. But if we are moving towards `it` we can rename these in core everywhere in a dedicated PR.